components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -56,6 +79,111 @@ export default class ConfirmIntegration extends React.Component {
         const outgoingHook = this.props.outgoingHooks[this.state.id];
         const oauthApp = this.props.oauthApps[this.state.id];
 
+        let copyLink = null;
+        if (document.queryCommandSupported('copy')) {
```

Comment:
Could we refactor this out into a common component so we're not repeating ourselves? It looks like the only thing that needs to change between instances of it is the link itself

Reply:
@hmhealey done please take another look and please let me know what do you think